### DISCUSSION TITLE
Lost my data

### DISCUSSION THREAD

**Discussion by rakshitha-ram** [excalidraw/excalidraw#6487] [General]
Hi Team,

I generally have found that exporting image on Google Chrome Browser doesn't work, which is why I use Safari to export images.
However today there were two issues that I found
1. My work created on Chrome browser didn't show up on Safari
2. When I reloaded Chrome, I lost the latest work that I had created
Can you please help me get back my latest work and also tell me why Chrome doesn't let me export.

**Comment by dwelle** [comment]
Hi.

We're storing data to the browser, so a Chrome browser and Safari browser will have its own data.

When you say you reloaded Chrome, what do you mean? If you reinstalled or cleared the browser data, then it would remove Excalidraw data as well :(.

As for exporting not working, what does it say or do when you try to export?
